src/http-client.ts

```diff
@@ -29,14 +29,30 @@ export class HttpClient {
     504
   ]);
 
+  static retryNetworkErrorCodes = new Set([
```

Comment:
There may be more network codes that we should retry on. These seem like the obvious three.